Tests: Add new behat tests for the new features in the homepage.

nader77 opened the pull request:
#521

HelenaEksler commented on capacity4more/behat/features/bootstrap/FeatureContext/Homepage.php:
```diff
@@ -30,6 +30,77 @@ public function iShouldSeeTheSiteHomepage() {
+    $steps[] = new Step\When('I should see "Featured block"');
+    $steps[] = new Step\When('I should see "Nobel Foundation" in the "div.pane-featured-block" element');
+    $steps[] = new Step\When('I should see "Nobel Banquet" in the "div.pane-featured-block" element');
+    $steps[] = new Step\When('I should see "Second Prize" in the "div.pane-featured-block" element');
```
@nader77 It looks not correct, because every time we visit the homepage - there will be 3 random featured items, not these three every time.

nader77 commented:
* Added new tests for the homepage new features.
* Fixed the suggested groups video block css.
* Fixed @amitaibu Comment in #510

@HelenaEksler PR is ready.

HelenaEksler commented:
@nader77 As to me it looks fine. Just waiting for tests become green.

nader77 commented:
@zero2one Ready for review.
